cpu/esp32/Makefile.dep

```diff
@@ -79,11 +75,6 @@ ifneq (,$(filter mtd,$(USEMODULE)))
   USEMODULE += esp_idf_spi_flash
 endif
 
-ifneq (,$(filter ndn-riot,$(USEPKG)))
```

Comment:
You removed this. Isn't this required anymore?

Reply:
I found this dependency strange that this dependency was only needed for this CPU. I think I tried finding out in the initial introduction why this was required but I could not find it. Why is this needed for esp32?

Reply:
> I found it strange that this dependency was only needed for this CPU.

Yes, but ESPs are the only CPUs that have an WiFi interface and use `wpa_supplicant` which comes with its own `crypto` module :wink:

When we added ESP32 support initially, there were symbol conflicts of `sys/crypto`/`sys/hashes` with `wpa_supplicant/crypto`. So we had to avoid that both `crypto` modules were used at the same time.

This fragment was only a part of the dependency resolution. `cpu/esp32/Makefile.dep` also included the following: https://github.com/RIOT-OS/RIOT/blob/3ac99877ac2939b087e7b639041c08a4ad9071fb/cpu/esp32/Makefile.dep#L29-L38.

With PR #11994 we solved the conflict by renaming the symbols in `wpa_supplicant` to something `wpa_<symbol>`. The part of the dependencies above were just forgotten to remove. So we can do this with this PR :+1:

Reply:
Ok, thanks for clarifying !